install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -256,6 +256,10 @@ data:
 {{- end }}
 {{- end }}
 
+{{- if .Values.global.externalIPs.enabled }}
+  enable-external-ips: "true"
```

Comment:
Don't we want to allow users to disable this feature? cilium-agent defaults to `false`.